### ISSUE TITLE
[BUG] Formatting multiple columns with summarizeFormat()

### ISSUE THREAD

**Issue by dansysanalyst** [Power-Components/livewire-powergrid#1529]
### Have you searched through other issues to see if your problem is already reported or has been fixed?

Yes, I did not find it.

### Did you read the documentation?

Yes, I did not find it.

### Have you tried to publish the views?

Yes - I didn't work.

### Is there an error in the console?

No

### Theme

Tailwind 3.x

### To Reproduce...

I would like to format the average summary in two different columns, one as a `kcal` and the other as `currency $`.

Expected:

![IMAGE 2024-05-05 11:06:47](https://github.com/Power-Components/livewire-powergrid/assets/79267265/db57467d-13f3-4701-a795-cddc9866b8c6)

--- 

Result:

![IMAGE 2024-05-05 11:06:49](https://github.com/Power-Components/livewire-powergrid/assets/79267265/c2fe67f9-8a50-4846-bfa9-cefcb9e4366e)

### Extra information

```php
Column::make('Calories', 'calories', 'calories')
                ->withAvg('Average', header: true, footer: false)
                ->sortable(),

Column::make('Price', 'price_formatted', 'price')
                ->withSum('Sum Price', header: true, footer: false)
                ->withAvg('Avg Price', header: true, footer: false)


```

```php
public function summarizeFormat(): array
{
    return [
        'calories.{avg}'          => fn ($value) => Number::format($value, locale: 'br') . ' kcal',
        'price.{sum,avg,min,max}' => fn ($value) => Number::currency($value, in: 'USD'),
        'price.{count}'           => fn ($value) => Number::format($value, locale: 'br'),
    ];
}
```

**Comment by dansysanalyst** [comment]
I should be sending a PR soon fixing this issue.

**Comment by dansysanalyst** [comment]
Fixed in [v5.6.0](https://github.com/Power-Components/livewire-powergrid/releases/tag/v5.6.0)
